browser_tests/fixtures/helpers/CommandHelper.ts

```
@@ -0,0 +1,68 @@
+    await this.page.evaluate(
+      ({ commandId, commandStr }) => {
+        const app = window.app!
+        const randomSuffix = Math.random().toString(36).substring(2, 8)
+        const extensionName = `TestExtension_${randomSuffix}`
+
+        app.registerExtension({
+          name: extensionName,
+          commands: [
+            {
+              id: commandId,
+              function: eval(commandStr)
```

Biome (2.3.13)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)